<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakeSpritesClass</class>
 <widget class="QDialog" name="MakeSpritesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MakeSprites</string>
  </property>
  <widget class="QLineEdit" name="pathLink">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>661</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Папка с графикой</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butPathLink">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butSave">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="info">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>761</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="butFileName">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>661</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выходной скрипт</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Эффекты</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butPathFX">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pathFX">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>661</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Программа</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="settingName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>661</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="makesprites.qrc"/>
 </resources>
 <connections/>
</ui>
